src/System.Net.Http/src/System.Net.Http.csproj

```diff
@@ -133,7 +133,20 @@
   </ItemGroup>
   
   <ItemGroup Condition=" '$(TargetsUnix)' == 'true' ">
-    <Compile Include="System\Net\Http\HttpClientHandler.Unix.cs" />
+    <Compile Include="System\Net\Http\Unix\HttpClientHandler.Unix.cs" />
+    <Compile Include="System\Net\Http\Unix\CurlHandler.cs" />
+    <Compile Include="$(CommonPath)\Interop\Unix\Interop.Libraries.cs">
+      <Link>System\Net\Http\Unix\Interop.Libraries.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonPath)\Interop\Unix\libcurl\Interop.libcurl.cs">
+      <Link>System\Net\Http\Unix\Interop.libcurl.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonPath)\Interop\Unix\libcurl\Interop.libcurl_types.cs">
+      <Link>System\Net\Http\Unix\Interop.libcurl_types.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonPath)\Interop\Unix\libcurl\Interop.SafeCurlHandle.cs">
+      <Link>System\Net\Http\Unix\Interop.SafeCurlHandle.cs</Link>
+    </Compile>
```

Comment:
For anything under CommonPath, the ```<Link>``` path should start with "Common\".  It also shouldn't have "System\Net\Http" in it.  So for example, the entry for Interop.Libraries.cs should look like:
```XML
<Compile Include="$(CommonPath)\Interop\Unix\Interop.Libraries.cs">
    <Link>Common\Interop\Unix\Interop.Libraries.cs</Link>
</Compile>
```
See other .csproj files like System.Console.csproj or System.IO.FileSystem.csproj for some examples.